app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Guest is not vaccinated")
+        elif datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("Vaccine is outdated")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("Guest without mask")
+        else:

```

Comment:
`else` is redundant and you can use `if` instead of `elif`

Reply:
Ok, thanks